### PR TITLE
Work on the slider + style

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: apps/festival/festival/src/app/marketplace/home/home.component.scss [hunk 1]
```diff
@@ -44,11 +44,15 @@
   background-color: var(--background-card);
 }
 
+movie-slide {
+  height: 700px;
```
Comment (fritzschoff) [inline apps/festival/festival/src/app/marketplace/home/home.component.scss:48]: maybe a good time make the height dynamic

File: libs/ui/src/lib/slider/slide/slide.component.scss [hunk 2]
```diff
@@ -9,13 +9,14 @@
...
   background-repeat: no-repeat;
   background-position: center;
-  @include frame();
+  height: fit-content;
+  // @include frame();
 
-  &-overlay {
-    @include frame();
-  }
+  // &-overlay {
```
Comment (fritzschoff) [inline libs/ui/src/lib/slider/slide/slide.component.scss:15]: remove or todo?

File: libs/ui/src/lib/slider/slider.component.scss [hunk 3]
```diff
@@ -1,36 +1,40 @@
...
-        display: flex;
-        position: relative;
-    }
+ul {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  // position: relative;
```
Comment (fritzschoff) [inline libs/ui/src/lib/slider/slider.component.scss:14]: remove then

File: libs/ui/src/lib/slider/slider.component.scss [hunk 4]
```diff
@@ -1,36 +1,40 @@
...
-        // TODO #2440
-        height: 600px;
-        display: flex;
-        position: relative;
-        flex-shrink: 0;
-    }
+li {
+  width: 100%;
+  display: flex;
+  // position: relative;
```
Comment (fritzschoff) [inline libs/ui/src/lib/slider/slider.component.scss:20]: same

File: libs/ui/src/lib/slider/slider.component.ts [hunk 5]
```diff
@@ -149,21 +121,15 @@ export class SliderComponent implements OnDestroy, AfterContentInit, AfterViewIn
...
   @ContentChildren(SlideComponent, { descendants: true }) slides: QueryList<SlideComponent>;
 
-  @ViewChild('wrapper') private slideWrapper: ElementRef<HTMLDivElement>;
-
   @ViewChild('slideList') private slideList: ElementRef<HTMLUListElement>;
 
   constructor(
-    private themeService: ThemeService,
+    @Inject(PLATFORM_ID) private platformId,
     private animationBuilder: AnimationBuilder,
```
Comment (fritzschoff) [inline libs/ui/src/lib/slider/slider.component.ts:128]: I still don't know why this `AnimationBuilder` is bad, but I guess you have no good words left for this, right?

**Review by fritzschoff — CHANGES_REQUESTED** [review]

File: apps/festival/festival/src/app/marketplace/home/home.component.scss (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/festival/festival/src/app/marketplace/home/home.component.scss:48]: I've checked and it actually work on every size until mobile. But on mobile we would want to change the image with the poster

File: libs/ui/src/lib/slider/slide/slide.component.scss (hunk 2 above)
Comment (GrandSchtroumpf) [reply libs/ui/src/lib/slider/slide/slide.component.scss:15]: good catch

File: libs/ui/src/lib/slider/slider.component.ts (hunk 5 above)
Comment (GrandSchtroumpf) [reply libs/ui/src/lib/slider/slider.component.ts:128]: It's not bad, it's just difficult to use. I didn't check it.

File: libs/ui/src/lib/slider/slider.interface.ts [hunk 6]
```diff
@@ -13,11 +13,9 @@ export interface Slider {
     hideIndicators: boolean;
 
     /** Appearance */
-    maxWidth: string;
-    maxHeight: string;
+    // maxWidth: string;
```
Comment (fritzschoff) [inline libs/ui/src/lib/slider/slider.interface.ts:16]: remove it

**Review by fritzschoff — APPROVED** [review]

**Pull request merged**